dev/annotate_source.py

```diff
@@ -138,18 +138,16 @@ class Annotator(Refresher):
   # regex for 'version-X.Y.Z' versions
   TAG_MATCHER = re.compile('^version-[0-9]+\.[0-9]+\.[0-9]+$')
 
-  def __init__(self, options):
+  def __init__(self, options, path=None):
     self.__next_tag = options.next_tag
-    self.__path = options.path
+    self.path = path or options.path
```

Comment:
If this was just to expose it, you could @property it with a getter.
The difference is that it would not be settable from the outside.